src/entrypoint.S

```diff
@@ -42,10 +42,13 @@ set_sp:
 	mtc0 v0,C0_SR
 	mtc0 $0,C0_CAUSE
 
-	/* Check if PI DMA transfer is required */
+	/* Check if PI DMA transfer is required,
+	   knowing that IPL3 loads 1MB of ROM to RAM,
+	   and __libdragon_text_start is located
+	   right after the ROM header where this 1MB starts. */
 	la a0, __libdragon_text_start
 	la a1, __data_end
-	li t0, 0x100000				/* stock IPL3 load size */
+	li t0, 0x100000				/* stock IPL3 load size (1MB) */
```

Comment:
We try to use IEC binary prefixes consistently in the documentation to avoid ambiguity, so this should be (1 MiB) instead. It is also recommended to put a space between the number and the unit. Same with the description comment above.